### ISSUE TITLE
Parsing partial time with "exact=True" returns a DateTime instead of Time

### ISSUE THREAD

**Issue by Delgan** [sdispater/pendulum#211]
Hi again.

```python
>>> import pendulum
>>> pendulum.parse("13:00:00", exact=True, strict=False)
Time(13, 0, 0)
>>> pendulum.parse("13:00", exact=True, strict=False)
DateTime(2018, 5, 15, 13, 0, 0, tzinfo=Timezone('UTC'))
```

I guess this should return a `Time` instance instead (it works as expected with partial dates).

**Comment by sdispater** [comment]
That's weird. But yes it should return a `Time` instance.

I will look into it.

**Comment by sdispater** [comment]
This should be fixed in the latest release: `2.0.2`
